src/exporters/MMDExporter.ts

```diff
@@ -1,18 +1,23 @@
-import { Matrix4, Quaternion, Vector3 } from 'three'
+import { Matrix4, Quaternion, Vector3, Bone, SkinnedMesh } from 'three'
+// @ts-ignore
 import { CharsetEncoder } from 'mmd-parser'
 
 /**
  * Dependencies
  *  - mmd-parser https://github.com/takahirox/mmd-parser
  */
 
-const MMDExporter = function () {
+interface CharsetEncoder {
+  s2uTable: { [key: string]: number }
+}
+
+const MMDExporter = function (): void {
```

Comment:
This should be turned into a class function

Reply:
Infact you could re-copy it from `three.js` because there might be some additions too 👀 and it should be a class already.

Reply:
> Infact you could re-copy it from three.js because there might be some additions too 👀 and it should be a class already.

Okay, I'll refer to `three.js` and fix it!